### PR TITLE
DRILL-7042: Fix deb package issues

### PR DESCRIPTION
**Pull request by nareshgbhat**
Fix the deb package issues
-Add apache as prefix
-Remove lib folder while packaging, because lib folder does not exist.

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

### CODE REVIEW

**Comment by nareshgbhat** [comment]
Thank you very much for review.  I will make the changes in next version, test and submit the patch ASAP.

**Comment by nareshgbhat** [comment]
@vdiravka I could not able to find logs directory. Am I missing anything ?

[centos@centos drill]$ ls distribution/target/apache-drill-1.15.0/apache-drill-1.15.0/
bin  conf  git.properties  jars  KEYS  LICENSE  NOTICE  README.md  sample-data  winutils
[centos@centos drill]$

**Comment by nareshgbhat** [comment]
When I update to the latest jdeb version https://mvnrepository.com/artifact/org.vafer/jdeb/1.7 the deb package creation is not happening. After "[DEBUG] -- end configuration --" directly jumps to "[INFO] Reactor Summary:" But with the 1.0 version it actually generates the deb package.  I am just looking into it.  Let me know if I am missing anything else to setup on project file.

drill$ mvn -X package -Pdeb -DskipTests
....
.................
[DEBUG] -- end configuration --
[INFO] No site descriptor found: nothing to attach.
[INFO] 
[INFO] --- jdeb:1.0:jdeb (default) @ distribution ---
[DEBUG] Dependency collection stats: {ConflictMarker.analyzeTime=52489, ConflictMarker.markTime=25408, ConflictMarker.nodeCount=82, ConflictIdSorter.graphTime=42869, ConflictIdSorter.topsortTime=28707, ConflictIdSorter.conflictIdCount=39, ConflictIdSorter.conflictIdCycleCount=0, ConflictResolver.totalTime=367722, ConflictResolver.conflictItemCount=78, DefaultDependencyCollector.collectTime=4902055, DefaultDependencyCollector.transformTime=561315}
[DEBUG] org.vafer:jdeb:jar:1.0:
....
[DEBUG] -- end configuration --
[INFO] Creating debian package: target/apache-drill-1.15.0.deb
[INFO] Building data

File: distribution/pom.xml [hunk 1]
```diff
@@ -441,26 +441,18 @@
                   <goal>jdeb</goal>
                 </goals>
                 <configuration>
-                  <deb>target/drill-${project.version}.deb</deb>
+                  <deb>target/apache-drill-${project.version}.deb</deb>
```
[the remark this answers is not in the document]
Comment (vdiravka) [reply distribution/pom.xml]: Add the following configs in order to update plugin version:
```
<skipPOMs>false</skipPOMs>
<deb>distribution/target/apache-drill-${project.version}/apache-drill-${project.version}.deb</deb>
```
see more: https://github.com/tcurdt/jdeb/blob/master/docs/maven.md

[the remark this answers is not in the document]
Comment (nareshgbhat) [reply distribution/pom.xml]: @vdiravka Thank you very much. I made the changes and the patch is ready. I am planning to test on a clean slate system both the DEB/RPM packages and push the patch for review hopefully by tomorrow.

**Comment by nareshgbhat** [comment]
My apologies for creating deb/rpm two separate pull requests.  I am closing both #1643 and #1645 both. Now on words please use the #1651 for review and feedback. I have tried to incorporate all your valuable suggested changes in my new PR

**Pull request closed without merging**